experimenter/experimenter/nimbus_ui_new/templates/nimbus_experiments/timeline.html

```diff
@@ -4,6 +4,7 @@
       <li class="list-group-item flex-fill text-center d-flex flex-column justify-content-center {% if status.is_active %}bg-primary text-white{% endif %}">
         <strong>{{ status.label }}</strong>
         <small>{{ status.date|default:'---' }}</small>
+        {% if status.days is not None %}<small>{{ status.days }} days</small>{% endif %}
```

Comment:
The current page has a little info tooltip about the experiment duration, not sure if it's really necessary but just wanted to flag the difference.

Reply:
Also, it looks like there's a built-in django template function for `pluralize` if you want to make `day` or `days` based on whether the value is plural.

Reply:
for all the tooltips- I have added a ticket to cover all the tooltips here is the ticket- https://github.com/mozilla/experimenter/issues/11748 and for the `pluralize` I have made that changes, good point though